cmd/kops/create_cluster.go

```diff
@@ -915,6 +917,9 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 					MaxRetries: fi.Int(3),
 				},
 			}
+			if c.OpenstackDNSServers != "" {
+				cluster.Spec.CloudConfig.Openstack.Router.DNSServers = fi.String(c.OpenstackDNSServers)
```

Comment:
router is not maybe correct place for this settings, subnet is the correct. However, i decided to not create new struct for this? What you think?

Reply:
I think this is more of an instance group property if we could sell this as a non-openstack change.

Reply:
I'm fine getting this working for just openstack.  As this is a subnet property I would imagine that the config might be in the InstanceGroup spec, turning the member `Subnets []string` to perhaps `Subnets []Subnet`.  A larger conversation there I believe.